board/opendingux/package/bennugd/Config.in

```diff
@@ -0,0 +1,14 @@
+config BR2_PACKAGE_BENNUGD
```

Comment:
Out of curiousity, what uses it?

Reply:
Streets of Rage Remake v5a and King of Fighters.
https://boards.dingoonity.org/gcw-releases/streets-of-rage-remake-v5a/
https://boards.dingoonity.org/gcw-releases/kof-flames-of-courage-v5/